src/driver/ExpressDriver.ts

```diff
@@ -61,12 +61,13 @@ export class ExpressDriver extends BaseDriver implements Driver {
      * Registers middleware that run before controller actions.
      */
     registerMiddleware(middleware: MiddlewareMetadata): void {
-        if (!middleware.instance.use)
+        const instance = middleware.instance;
```

Comment:
are such changes really increases performance so much? Also good to have a benchmark results